src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -1685,6 +1720,23 @@ public static void loadGameCharacterVariablesFromXML(GameCharacter character, St
 			}
 		}
 		
+		// ************** Player Slavery **************//
+		
+		nodes = parentElement.getElementsByTagName("playerSlavery");
+		Element playerSlaveryElement = (Element) nodes.item(0);
+		if(playerSlaveryElement!=null) {
+			element = (Element) playerSlaveryElement.getElementsByTagName("rules").item(0);
+			
+			if(element.getElementsByTagName("rule").getLength() > 0)
```

Comment:
assign element.getElementsByTagName("rule") to a separate variable.

Reply:
Handled